### DISCUSSION TITLE
Installing the module using .so and not the RPM dependencies

### DISCUSSION THREAD

**Discussion by Amaury4sg** [OpenIDC/mod_auth_openidc#987] [Q&A]
Hello,
in our entreprise we don't install the apache on the default location (it's not always in the same path)
We are looking to do a standard installation procedure
Should we do a yum install with the official package => the modules (and a new apache) will be installed on the default location.
We don't want to compile the module each time, we'd like to avoid needing the root privilege on the installation as well.

Is it possible to copy the libraries manually (should we have a .so set per RHEL version ?)
for example
LoadModule libjansson_module modules/libjansson.so.4
LoadModule libcjose_module modules/libcjose.so.0
LoadModule auth_openidc_module modules/mod_auth_openidc.so
(we won't use the reddis cache)

And keep the support.

Thanks for your help

**Comment by brandonk10** [comment]
Probably not a module question, but check out [this question](https://superuser.com/questions/160530/override-rpm-install-path) .  It might give you a procedure, and also talks about unpacking the rpm so you can see the files.  You should also look at dependent rpms(rpm -q --requires).

**Reply by Amaury4sg** [reply]
Thanks a lot for your time brandonk10 !
yep we did open the RPM to list the dependencies,
The question was "is this practice supported by zmartzone"
And if that's the case should we take one .so file per OS ?
Some .so file can be used in RHEL (or Centos) 6,7 and 8
